content/docs/overview.md

```diff
@@ -172,15 +172,15 @@ main :: proc() {
 }
 ```
 
-The `core:` prefix is to state where the import is meant to look; this is called a library collection. If no prefix is present, the import will look relative to current file.
+The `core:` prefix is used to state where the import is meant to look; this is called a library collection. If no prefix is present, the import will look relative to the current file.
 
 
-**Note**: By convention, the package name is the same as the last element in the import path. `"core:fmt"` package comprises of files that begin with the statement `package fmt`. However, this is not enforced by the compiler, which means that default name for the import name will be determined by the last element in the import path if possible.
```

Comment:
`import name` is correct in this case to emphasise the entity name itself rather than the "operation".

Reply:
Ok, will revert this change.